### PR TITLE
refactor: UI improvements

### PR DESCRIPTION
**Pull request by csosamorales**
Related tasks:
- add paste button on Owner Address field (UI12)
- add Site Card pointer onHover (UI13)
- add tooltip on attributes card (UI14)
- add icons on wallet button

### CODE REVIEW

**Comment by csosamorales** [comment]
image

image

feat_ UI improvements by csosamorales · Pull Request  67 · fleekxyz_non-fungible-apps

**Review by zoruka — CHANGES_REQUESTED** [review]
Left some comments of possible improvements. For a latter instance I think we should at all avoid using `px` units to defining sizes all of them should be replaced for `rem` as it makes really easier to improve the application responsiviness. Also is important to notice that all the hardcoded colors should be switched to a theming file following the UI framework suitability like here https://chakra-ui.com/docs/styled-system/customize-theme. These changes are the point to make our app more maintainable as the project grows, almost every time that some kind of information is repeated seems like it should be abstracted away so changes can be done easily with a single source of truth.

**Comment by zoruka** [comment]
Another comment, just to keep patterns I think this PR should be called `refactor: ...` as it is not adding new features but improving the current ones.

File: ui/src/components/wallet-button/wallet-icons.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { AiOutlineCopy } from 'react-icons/ai';
+import Metamask from 'assets/MetaMask.png';
+import { WalletType } from '.';
+
+type ConnectedWalletIconProps = {
+  provider: string;
+};
+
+export const WalletIcon = () => {
+  return <Icon as={FaWallet} />;
```
[the remark this answers is not in the document]
Comment (csosamorales) [reply ui/src/components/wallet-button/wallet-icons.tsx:13]: cool thanks! I had something similar on mind and that example helps me a lot 🙏🏼

File: ui/src/components/wallet-button/wallet-icons.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { Icon, Image } from '@chakra-ui/react';
+import { FaWallet } from 'react-icons/fa';
+import { IoExitOutline } from 'react-icons/io5';
+import { AiOutlineCopy } from 'react-icons/ai';
+import Metamask from 'assets/MetaMask.png';
```
[the remark this answers is not in the document]
Comment (csosamorales) [reply ui/src/components/wallet-button/wallet-icons.tsx:5]: I agree about the SVG, but didn't find any metamask SVG file 😞

File: ui/src/views/mint-site/mint-site.tsx [hunk 3]
```diff
@@ -42,6 +44,31 @@ const initialValues = {
...
+        <Button
+          h="1.75rem"
+          size="sm"
+          onClick={() => {
+            if (setFieldValue && props.name) {
+              navigator.clipboard
+                .readText()
+                .then((text) => setFieldValue(props.name as string, text));
+            }
+          }}
```
Comment (zoruka) [inline ui/src/views/mint-site/mint-site.tsx:63]: Let's avoid adding functions in the middle of the component render to improve readability.

**Review by zoruka — CHANGES_REQUESTED** [review]
Overall the implementations looks good to me. Just a small change request.

**Review by zoruka — APPROVED** [review]

**Pull request merged**